server/models/ArticleModels.js

```diff
@@ -11,18 +11,48 @@ function fetchArticleById(id) {
     });
 }
 
-function fetchArticles() {
-  return db
-    .query(
-      `SELECT articles.*, COUNT(comment_id) AS comment_count 
-  FROM articles
-  LEFT JOIN comments ON comments.article_id = articles.article_id
-  GROUP BY articles.article_id
-  ORDER BY created_at DESC`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+function fetchArticles(topic, sort_by = "created_at", order_by = "DESC") {
+  if (
+    sort_by &&
+    sort_by !== "author" &&
+    sort_by !== "title" &&
+    sort_by !== "article_id" &&
+    sort_by !== "topic" &&
+    sort_by !== "created_at" &&
+    sort_by !== "votes" &&
+    sort_by !== "comment_count"
```

Comment:
you can put these values in an array then use 
`![ "author", "title", ...etc ].includes(sort_by)` 
to check whether `sort_by` is one of your greenlisted values